docs/standard/frameworks.md

```diff
@@ -1,56 +1,293 @@
 
-There are multiple ways to refer to frameworks in written form, most of which are used in this documentation. They are described below, both as a legend for interpreting the documentation but also to guide use in other documents.
+A framework is typically referenced by a short target framework moniker or *TFM*. In .NET Standard, this is also generalized to *TxM* to allow a single reference to multiple frameworks. The NuGet clients support the following frameworks. Equivalents are shown within brackets (`[]`).
 
-Using .NET Framework 4.6.1 as an example, the following forms can be used:
+Name | Abbreviation | TFMs/TxMs
+--- | --- | ---
+.NET Framework | net | net11
+ | | net20
+ | | net35
+ | | net40
+ | | net403
+ | | net45
+ | | net451
+ | | net452
+ | | net46
+ | | net461
+ | | net462
+.NET Core | netcore | netcore [netcore45]
+ | | netcore45 [win, win8]
+ | | netcore451 [win81]
+ | | netcore50
+.NET MicroFramework | netmf | netmf
+Windows | win | win [win8, netcore45]
+ | | win8 [netcore45, win]
+ | | win81 [netcore451]
+ | | win10 (not supported by Windows 10 platform)
+Silverlight | sl | sl4
+ | | sl5
+Windows Phone | wp | wp [wp7]
+ | | wp7
+ | | wp75
+ | | wp8
+ | | wp81
+ | | wpa81
+Universal Windows Platform | uap | uap [uap10.0]
+ | | uap10.0
+.NET Standard | netstandard | netstandard1.0
+ | | netstandard1.1
+ | | netstandard1.2
+ | | netstandard1.3
+ | | netstandard1.4
+ | | netstandard1.5
+ | | netstandard1.6
+.NET Core App | netcoreapp | netcoreapp1.0
 
-**Referring to a product**
+## Deprecated frameworks
 
-You can refer to a .NET platform or runtime.
+The following frameworks are deprecated. Packages targeting these frameworks should migrate to the indicated replacements.
 
-- ".NET Framework 4.6.1"
+Deprecated framework | Replacement
+--- | ---
+aspnet50 | netcoreapp
+aspnetcore50 | 
+dnxcore50 | 
+dnx | 
+dnx45 | 
+dnx451 | 
+dnx452 | 
+dotnet | netstandard
+dotnet50 | 
+dotnet51 | 
+dotnet52 | 
+dotnet53 | 
+dotnet54 | 
+dotnet55 | 
+dotnet56 | 
+winrt | win
 
-**Referring to a Framework**
+## Precedence
 
-You can refer to a framework or targeting of a framework using long- or short-forms of the TFM. Both are equally valid in the general case.
+A number of frameworks are related to and compatible with one another but not necessarily equivalent:
 
-- `.NETFramework,Version=4.6.1`
-- `net461`
+Framework | Can use
+--- | ---
+uap (Universal Windows Platform) | win81
+| | wpa81
+| | netcore50
+win (Windows Store) | winrt
+| | winrt45
 
-**Referring to a family of Frameworks**
+## .NET Standard
 
-You can refer to a family of frameworks using long- or short-forms of the framework ID. Both are equally valid in the general case.
+The [.NET Standard](https://github.com/dotnet/standard) simplifies references between binary-compatible frameworks, allowing a single target framework to reference a combination of others. For background, see the [.NET Platform Guide](https://docs.microsoft.com/dotnet/articles/standard/index).
```

Comment:
use relative link to topics inside the repo. and probably a better link is this: https://docs.microsoft.com/en-us/dotnet/articles/standard/library. I'll submit a PR to the NuGet docs for this.